classes/person.rb

```diff
@@ -21,6 +22,10 @@ def can_use_services?
     is_of_age? || @parent_permission
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Great job so far, in order to achieve the` has-many` association the person itself needs to be sent. Kindly fix that.

`i.e  Rental.new(....)`

Reply:
Done